CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## In Development
+* Bump to latest CircleCI orb versions (by @ZoeLeah)
+* Remove unsupported k8s Versions (by @ZoeLeah)
```

Comment:
Please mention explicitly (eg. 26,27,28) the new K8s versions that are used for testing this Helm chart.
That would provide a better info for someone reading the Changelog and thinking if particular K8s version is supported or not.